keras_cv/models/segmentation/segment_anything/sam.py

```diff
@@ -125,9 +119,8 @@ class SegmentAnythingModel(Task):
     (i.e. `masks[:, 1:, ...]`) are alternate predictions that can be used if
     they are desired over the first one.
 
-    Now, in case of only points and box prompts, do:
+    Now, in case of only points and box prompts, simply exclude the masks:
 
-    >>> no_input_mask = np.empty((1, 0, 256, 256, 1))
```

Comment:
I think you forgot to remove the `no_mask_input` key from the input dictionary below.

Reply:
Good catch -- thanks!